gradle.properties

```diff
@@ -18,7 +18,7 @@
 # org.gradle.parallel=true
 
 VERSION_CODE=16
-VERSION_NAME=1.9.0-SNAPSHOT
+VERSION_NAME=2.1.1-SNAPSHOT
```

Comment:
Why such a big jump? Guessing because of the internal versions published or something?

Reply:
@BenSchwab No, the previous version updates were done on release branches and never merged back to master.